qabelbox/src/main/java/de/qabel/qabelbox/account/AccountManager.java

```diff
@@ -1,80 +0,0 @@
-package de.qabel.qabelbox.account;
```

Comment:
I just removed https://github.com/Qabel/qabel-android/pull/728/files#diff-82860000c643901cc1b744a460f92342L70 to refresh quota in background when calling getQuota. If the quota changed the manager sents an intent.